tutorials/Intro-to-geospatial-analysis/index

```diff
@@ -0,0 +1,426 @@
+---
+title: An Introduction to Geospatial Analysis Using Google Earth Engine
+description: This workshop provides a fundamental overview of remote sensing and geospatial analysis using the JavaScript API of Earth Engine.
+author: TC25
+tags: introductory, GIS, remote sensing, Google Earth Engine
+date_published: 2019-09-14
+---
```

Comment:
Hi TC! What do all the "+" prefixes mean here and throughout? Were you meaning to put the whole content in a table?

Reply:
There is no + in the actual markdown. I think it just indicates whether something was added to the file.